governance.md

```diff
@@ -32,33 +32,64 @@ Every code owner is expected to:
 
->**NOTE:** Only the ultimate decision makers can request to revisit the decision before the revision date. To do it, raise an issue in the `community` repository with all details and the reason why you want to discuss the decision earlier than agreed.
+### Reach a consensus
+
+Take all necessary actions to reach a consensus no later than the decision due date:
+
+- Send a link to the **Decision record** issue to the related SIG or WG mailing list and post it in the relevant Slack channels. If people relevant for the decision are not part of the mailing list, add them explicitly to the email communication. Communicate the decision to be made clearly to the groups affected by it and invite them to check the proposal. 
+
+- Clarify and discuss the decision content and the proposal as needed. Use the mailing list, relevant Slack channels, the related pull request, or comment directly on the **Decision record** issue. 
+
+- Feel free to communicate the decision proposal during the upcoming SIG or WG meeting and ask its members for feedback. Encourage the discussion and bring up any objections early in the process. 
+
+- Those who created the proposal work with those who had objections to either prepare an improved solution or decline the proposal. 
+
+> **NOTE:** Discussions  lead to changes in the decision record or the proposal, or end up with no changes required. If someone suggests a substantially different approach, ask its supporters to write a counter proposal and to submit it in a separate pull request. 
+
+
+### Close the decision
+
+Once you reach the consensus:
+
+- Add the status change in the **Decision record** issue with either `Accepted` or `Declined`. 
+- Close and merge the pull request with the accepted proposal. Make sure that the merge comment contains one of the [keywords](https://help.github.com/articles/closing-issues-using-keywords/) to automatically close the **Decision record** issue. Otherwise, close the **Decision record** issue manually.
+
+> **NOTE:** If there are any pull requests with counter proposals to the decision record, close the related pull requests with rejected proposals without merging. State the reasons for the rejection in the closing comments.
+
+### Lack of consensus
+
+Engage the ultimate decision makers if there are still unresolved objections by the decision due date. The ultimate decision makers are stated in the decision log of the **Decision record** issue. Work with them to reach the final decision as soon as possible and close the issue.
+
+### Revisit a decision
+
+Raise an explicit request to revisit a decision or to review it. To request revision:
+
+- Create a new  decision record in the respective repository and populate the `Affected decisions` parameter. 
+- Add the  `Due date` parameter accordingly to the earliest revisit date of the affected decision. 
```

Comment:
I don't quite understand that point - does it mean ``Provide the new revision date in the`Due date` parameter.`` ? (there was also an additional space)

Reply:
@natasab : This means that the due date of the new (revised) decision regularly cannot be before 
the earliest revisit date of the original decision.
 @kazydek proposed: Specify the `Due date` parameter and add a new revision date that cannot be earlier than the revision date of the original decision
@natasab , @kazydek : agreed on reformulation